helmfile/values/networkpolicies/service/harbor.yaml.gotmpl

```diff
@@ -100,6 +100,24 @@ rules:
     {{ template "old-style.rule.gen" . }}
   {{ end }}
 
+  {{- if eq .Values.harbor.persistence.type "swift" }}
+  egress-rule-swift-object-storage:
```

Comment:
Can we not just override the `egress-rule-object-storage` and remove the if further down?
Also you should be able to use `old-style.rule.gen` for this.
And you should not need to add `objectStorage` ips with the new upgrade-ips script.